optimize/components/userguide/edit-mode.md

```diff
@@ -34,7 +34,11 @@ In the **Add a Tile** modal, choose **Text** to access the text editor. This ena
 
 ![text editor](./img/dashboard-addAReportModal-textReport.png)
 
-### Placing tile on the dashboard
+## Copying, Editing, and Removing Tiles
```

Comment:
Grammatical note to keep this sentence case, so "Copying, editing, and removing titles"

Reply:
oh, sorry, my bad